modules/kubernetes/aks-core/k8s-cluster-role.tf

```diff
@@ -20,24 +26,30 @@ resource "kubernetes_cluster_role" "helm_release" {
   }
 }
 
-resource "kubernetes_cluster_role" "toolkit_helm_release" {
+resource "kubernetes_cluster_role" "toolkit_edit" {
   metadata {
-    name = "toolkit-helm-release"
+    name = "toolkit-edit"
```

Comment:
"Toolkit" is a fairly common word. Given that this is global naming, is it unique enough?

Reply:
I put all of them together and updated the name to custom_cr_edit